tests/helpers/test_network.py

```diff
@@ -303,23 +286,13 @@ async def test_get_external_url_cloud_fallback(hass: HomeAssistant):
         "async_remote_ui_url",
         return_value="https://example.nabu.casa",
     ):
-        assert _async_get_external_url(hass, allow_cloud=False) == "http://1.1.1.1:8123"
-        assert (
-            _async_get_external_url(hass, allow_ip=False) == "https://example.nabu.casa"
-        )
+        assert _get_external_url(hass, allow_cloud=False) == "http://1.1.1.1:8123"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)